packages/@ourworldindata/grapher/src/entitySelector/EntitySelector.tsx

```diff
@@ -1330,6 +1330,7 @@ export class EntitySelector extends React.Component<{
                     placeholder={`Search for ${a(this.searchPlaceholderEntityType)}`}
                     trackNote="entity_selector_search"
                     onKeyDown={this.onSearchKeyDown}
+                    autoFocus
```

Comment:
The `EntitySelector` component has an `autoFocus` prop that properly got lost in a recent refactor. Can you use that here so that `autoFocus` is set only when it should be?